file-formats/uist/uist-v1.json

```
@@ -0,0 +1,134 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/uist/uist-v1.json",
+  "title": "Fiori Launchpad Space Template",
+  "description": "Fiori Launchpad Space Template",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "Format version",
+      "description": "Format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 100
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP for Key Users",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP for key user extensibility",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "spaceTemplateId": {
+      "title": "ID",
+      "description": "ID",
+      "type": "string",
+      "maxLength": 35
+    },
+    "spaceTemplateTitle": {
+      "title": "Title",
+      "description": "Title",
+      "type": "string",
+      "maxLength": 100
+    },
+    "spaceTemplateDescription": {
+      "title": "Description",
+      "description": "Description",
+      "type": "string",
+      "maxLength": 100
+    },
+    "spaceTemplateSortPriority": {
+      "title": "Sort Priority",
+      "description": "Sort priority",
+      "type": "number",
+      "minimum": 0.001,
+      "maximum": 999.999,
+      "multipleOf": 0.001
```

Most probably, the automatic generation will regenerate the property `"multipleOf"`to your schema. We plan to fix this.

```suggestion
      "maximum": 999.999
```

---

I think we have to remove the `multipleOf` manually, currently. We plan to solve this in the generator soon.

@larshp Or do you think we shall keep it for now (until the generator is fixed) otherwise the validation will raise an issue, won't it?